### PR TITLE
[BB-3584] Add explanation to deployment page 

### PR DESCRIPTION
**Pull request by spokerman12**
**Description**

We're fulfilling the following user story:

> "As a OCIM user, I want to know why deployment is taking so much time and information regarding that so that I am aware of the complexities"

**Supporting information**

- [Jira ticket](https://tasks.opencraft.com/browse/BB-3584)
- [Blogpost draft](https://docs.google.com/document/d/11hlSn40RKN82ePtxs0e5j0CwgM5POhFHwdopoXalMFA/edit?usp=sharing)

**Testing instructions**

1. Set up a local OCIM environment using this branch
2. Make the necessary changes (most likely on `ui/components/Header`) to bypass the login logic. 
3. Head to `http://localhost:3000/console/notice`

**Snapshots:**

image

Also, check Jira ticket.

**Further considerations:**

- Seeing that I'm not fully aware of the roadmap, I could use some feedback on the blogpost
- At some point we have to update fonts for most components. 

Reviewer:
[ ] @Agrendalath

### CODE REVIEW

File: frontend/src/console/components/NoticeBoard/displayMessages.ts [hunk 1]
```diff
@@ -25,8 +25,24 @@ const messages = {
...
+  explanation: {
+    defaultMessage:
+      'There are many factors that affect the speed at which updates will ' +
+      'reflect on your site. Please allow up to 2 hours for your site to be updated.',
+    description: 'Brief explanation on deployment duration.'
+  },
+  blogpost_text: {
+    defaultMessage:
+      "If you'd like to read more about the way we provision the sites, " +
+      'you can read our blog post about the process ',
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/displayMessages.ts:37]: Was this text on the designs? Wouldn't it sound better like this?
```js
"If you would like to learn more about the way we provision the sites, please see our <link>blog post</link> explaining the deployment process."
```

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx [hunk 2]
```diff
@@ -194,15 +194,30 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
     </p>
   );
 
+  const staticExplanation = (
+    <div className="explanation-container">
+      <p>
+        <WrappedMessage id="explanation" messages={messages} />
+        <br />
+        <br />
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:202]: Please see the explanation of the `br` element usage in the [HTML spec](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element). I believe it falls into the second category in this scenario.

TL;DR: please change it to:
```html
</p>
<p>
```

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx [hunk 3]
```diff
@@ -194,15 +194,30 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
...
+        <WrappedMessage id="blogpost_text" messages={messages} />
+        <a href="https://opencraft.com/blog">
+          <WrappedMessage id="blogpost_link" messages={messages} />
+        </a>
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:206]: Let's use the following approach (along with the suggestion from `NoticeBoard/displayMessages.ts`:
```js
// TODO: Point the link to a specific blog post, once it's created.
<WrappedMessage
  id="blogpost_text"
  messages={messages}
  values={{
    link: (text: string) => (
      <a href="https://opencraft.com/blog">{text}</a>
    )
  }}
/>
```

File: frontend/src/console/components/NoticeBoard/displayMessages.ts [hunk 4]
```diff
@@ -25,8 +25,24 @@ const messages = {
...
+    description: 'Text that links to the blog post.'
+  },
+  blogpost_link: {
+    defaultMessage: 'here ',
+    description: 'Link to the blog post.'
+  },
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/displayMessages.ts:43]: I suppose this will not be needed.

File: frontend/src/console/components/NoticeBoard/displayMessages.ts [hunk 5]
```diff
@@ -25,8 +25,24 @@ const messages = {
...
   noticeBoard: {
-    defaultMessage: 'Notice Board',
+    defaultMessage: 'Status & Notifications',
     description: 'Notice board heading.'
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/displayMessages.ts:46]: Is this still accurate?

File: frontend/src/console/components/NoticeBoard/styles.scss [hunk 6]
```diff
@@ -40,6 +40,17 @@
 }
 
 .notice-board-heading {
-  font-size: 25px;
+  font-family: $chivo-font;
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/styles.scss:43]: Has the proper font family been settled out? I've seen your comment in Jira, but I could only find the Zeplin designs, which contained `Montserrat`, hence the question.

**Review by Agrendalath — CHANGES_REQUESTED** [review]

**Comment by Agrendalath** [comment]
@spokerman12, also a general suggestion - please use the [imperative mood](https://chris.beams.io/posts/git-commit/#imperative) in the commit's subject title.

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx (hunk 2 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:202]: Aha! Thanks for that! Added as requested.

File: frontend/src/console/components/NoticeBoard/displayMessages.ts (hunk 1 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/displayMessages.ts:37]: It's not, but it didn't mention the blogpost either. I like your suggestion. Added as such.

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx (hunk 3 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:206]: I'll keep this one in mind! Thanks! Applied.

File: frontend/src/console/components/NoticeBoard/displayMessages.ts (hunk 5 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/displayMessages.ts:46]: Nope. Changed.

File: frontend/src/console/components/NoticeBoard/styles.scss (hunk 6 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/styles.scss:43]: Yep, it's Chivo. The one shown in the mockups is not correct, as discussed [here](https://tasks.opencraft.com/browse/BB-3500?focusedCommentId=187842&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-187842)

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx [hunk 7]
```diff
@@ -198,12 +198,17 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
     <div className="explanation-container">
       <p>
         <WrappedMessage id="explanation" messages={messages} />
-        <br />
-        <br />
-        <WrappedMessage id="blogpost_text" messages={messages} />
-        <a href="https://opencraft.com/blog">
-          <WrappedMessage id="blogpost_link" messages={messages} />
-        </a>
+        <p />
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:201]: Could we just separate these into two paragraphs? It looks odd to have an empty paragraph inside a paragraph.

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx [hunk 8]
```diff
@@ -198,12 +198,17 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
...
+        {/* TODO: Point the link to a specific blog post, once it's created. */}
+        <WrappedMessage
+          id="blogpost_text"
+          messages={messages}
+          values={{
+            link: (text: string) => (
+              <a href="https://opencraft.com/blog">{text}</a>
```
Comment (Agrendalath) [inline frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:208]: I forgot about the `target="_blank" rel="noopener noreferrer"` here. Please add it.

**Review by Agrendalath — APPROVED** [review]
:+1:
* [x]  I tested this: tested this locally
* [x]  I read through the code
* [x]  I checked for accessibility issues
* [x]  Includes documentation: n/a

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx (hunk 7 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:201]: That was the linter. Sure!

File: frontend/src/console/components/NoticeBoard/NoticeBoard.tsx (hunk 8 above)
Comment (spokerman12) [reply frontend/src/console/components/NoticeBoard/NoticeBoard.tsx:208]: I should've picked it up, too. Added
